src/main/kotlin/com/vk/admstorm/highlight/markers/ServiceGutterIconNavigationHandler.kt

```diff
@@ -43,10 +43,19 @@ class ServiceGutterIconNavigationHandler(private val keyName: String, private va
 
         executeOnPooledThread {
             val response = service.execCommand(keyName)
-
             val popup = when {
                 response.errorMessage != null -> {
-                    service.errorMessage(response.errorMessage)
+                    val limitWordLength = 15
+                    val firstQuoteIndex = response.errorMessage.indexOfFirst { it == '\'' }
+                    val lastQuoteIndex = response.errorMessage.indexOfLast { it == '\'' }
+
+                    if (lastQuoteIndex - firstQuoteIndex > limitWordLength) {
```

Comment:
I understand you want to process the key length. Currently, the maximum key length is 100 characters. Can't she fit in?